content/docs/intro/cloud-providers/kubernetes/_index.md

```diff
@@ -56,12 +56,14 @@ The following SDKs are available to work with IaaS resources, and managed or sel
 
 The [`pulumi/kubernetes`](https://github.com/pulumi/pulumi-kubernetes) SDK is available to work with, and deploy app workloads to running Kubernetes clusters:
 
+<!-- markdownlint-disable ul-indent -->
```

Comment:
Do we need to impose this linting rule?

Reply:
(I mean, can we just remove the rule itself, since it seems reasonable to need to be able to nest Markdown list items.)

Reply:
@cnunciato I guess not. I don't really have a strong opinion about it. Happy to add it to the global ignores.

Reply:
Ok cool. Yeah, I'd probably lean toward just allowing that -- again, since there's no real reason not to (that I can think of).